packages/web-react/src/components/Tooltip/useFloating.ts

```diff
@@ -125,7 +128,14 @@ export const useFloating = (props: UseTooltipUIProps) => {
 
   // Floating UI library interaction hooks
   const click = useClick(context, { enabled: isClickEnabled });
-  const hover = useHover(context, { enabled: isHoverEnabled });
+  const hover = useHover(context, {
```

Comment:
Please, make this a custom hook with naming based on the prop, e.g. `useFocusableOnHover` or something similar.

Reply:
Floating UI does not export all of its types, so it was a problem to create a hook in a different file with imports from their library.

Thus, I created a function in this file to make it more readable.